lib/modules/datasource/maven/__snapshots__/index.spec.ts.snap

```diff
@@ -190,3 +190,75 @@ exports[`modules/datasource/maven/index skips registry with invalid metadata str
   ],
 }
 `;
+
+exports[`modules/datasource/maven/index supports artifactregistry urls with auth 1`] = `
```

Comment:
Please inline these two fixtures in `index.spec.ts`, as snapshots are not needed here and would be harder to maintain. Otherwise LGTM.

Reply:
Hi @Churro,

Do you have an example of what exactly you mean? A link to some existing code should be good enough.

Reply:
Sure, so instead of calling `Fixtures.get('pom.xml')`, simply embed the expected result. Similar to here:

https://github.com/renovatebot/renovate/blob/bec2dbf66517179d87a7a7112f695c24d83f3efe/lib/modules/datasource/maven/index.spec.ts#L266-L279